templates/payg/algolia.mdx

```diff
@@ -2,4 +2,279 @@
+    At the end of the billing period, Lago will calculate the sum of requests for each metric using the value of each event (i.e. the event property) and reset the amount to 0 as the aggregation type is metered.
+
+    <Frame>
+    <img src="/templates/images/algolia-aggregate-requests.png" />
+    </Frame>
+  </Tab>
+  <Tab title="API">
+  ```bash Create billable metrics
+      LAGO_URL="https://api.getlago.com"
+      API_KEY="__YOUR_API_KEY__"
+
+      curl --location --request POST "$LAGO_URL/api/v1/billable_metrics" \
+      --header "Authorization: Bearer $API_KEY" \
+      --header 'Content-Type: application/json' \
+      --data-raw '{
+        "billable_metric": {
+          "name": "Search Requests",
+          "code": "search_requests",
+          "description": "Requests made via the Search API",
+          "aggregation_type": "sum_agg",
+          "weighted_interval": null,
+          "recurring": false,
+          "field_name": "search_requests_volume",
+          "filters": []
+        }
+      }
+ 
+  ```
+  </Tab>
+</Tabs>
+
+## Step 2: Create a plan with multiple charges
+Now that we have all our billable metrics, we can replicate Algolia’s pricing. As this is a ‘pay-as-you-go’ plan, there is no subscription fee. In the plan settings, we can simply define a monthly billing period and then add our charges.
+
+<Tabs>
+  <Tab title="Dashboard">
+    For both charges, we use the **package pricing model**. This model allows us to create packages of units and define a unique package price. Customers will be charged for usage based on packages of 1,000 requests. Algolia offers 10,000 requests for free each month, for both APIs. Beyond this limit, each ‘opened’ package is due, which means that 800 Search requests cost the same price as 1,000 requests (i.e. $1.50).
+    At the end of each billing period (i.e. each month), Lago will automatically invoice each customer according to their consumption, then all counts and calculations will be reset to zero for the next period.
+
+    <Frame>
+      <img src="/templates/images/algolia-package-pricing.png" />
+    </Frame>
+  </Tab>
+
+  <Tab title="API">
+  ```bash Create a plan with multiple charges
+      LAGO_URL="https://api.getlago.com"
+      API_KEY="__YOUR_API_KEY__"
+      curl --location --request POST "$LAGO_URL/api/v1/plans" \
+      --header "Authorization: Bearer $API_KEY" \
+      --header 'Content-Type: application/json' \
+      --data-raw '{
+       "plan": {
+            "name": "Algolia Starter",
+            "code": "algolia_starter",
+            "interval": "monthly",
+            "amount_cents": 0,
+            "amount_currency": "USD",
+            "trial_period": 0.0,
+            "pay_in_advance": false,
+            "charges": [
+                {
+                    "lago_billable_metric_id": "15879740-102e-43bd-a6bc-15d865bb1b85",
```

Comment:
This field should be billable_metric_id when POSTing the request @AlexEven

Reply:
Just updated it, thank you Raffi!